src/language/LanguageManager.js

```diff
@@ -449,60 +455,70 @@ define(function (require, exports, module) {
 
     /**
      * Identifier for this language
+     * @private
      * @type {string}
      */
     Language.prototype._id = null;
 
     /**
      * Human-readable name of this language
+     * @private
      * @type {string}
      */
     Language.prototype._name = null;
 
     /**
      * CodeMirror mode for this language
+     * @private
      * @type {string}
      */
     Language.prototype._mode = null;
 
     /**
      * File extensions that use this language
+     * @private
      * @type {Array.<string>}
      */
     Language.prototype._fileExtensions = null;
 
     /**
      * File names for extensionless files that use this language
+     * @private
      * @type {Array.<string>}
      */
     Language.prototype._fileNames = null;
 
     /**
      * Line comment syntax
+     * @private
      * @type {Array.<string>}
      */
     Language.prototype._lineCommentSyntax = null;
 
     /**
      * Which language to use for what CodeMirror mode
+     * @private
      * @type {Object.<string,Language>}
      */
     Language.prototype._modeToLanguageMap = null;
 
     /**
      * Block comment syntax
+     * @private
      * @type {{ prefix: string, suffix: string }}
      */
     Language.prototype._blockCommentSyntax = null;
 
     /**
      * Whether or not the language is binary
+     * @private
      * @type {boolean}
      */
     Language.prototype._isBinary = false;
 
     /**
      * Returns the identifier for this language.
+     * @private
```

Comment:
Language is effectively exported by `defineLanguage` API which returns a Language Object. So The Language Object needs to be treated as a public API. Modify all private usages as public that are not `_vars`.